user-registration-app/src/app/views/user-profile-view/user-profile-view.component.css

```diff
@@ -92,19 +92,40 @@
 }
 
 .info-section {
-  padding:          2rem;
+  padding: 2rem;
   flex-basis: auto;
   background-color: #333333;
 }
 
+.collapsible-header {
+  background-color: #333333;
+  border-bottom: 1px solid #666666;
+  padding: 1rem 0 1rem 0;
+  justify-content: space-between;
+}
+
+.collapsible-body {
+  border: none;
+}
+
+.collapsible {
+  border: none;
+  border-top: 1px solid #666666;
+  box-shadow: none;
+  display: block;
+}
+
+.flex {
+  display: flex;
+}
+
```

Comment:
I'm pretty shit at css, so if someone else wants to make this better, go right ahead.

Reply:
is anyone actually good at css or are they really just good at googling css?

Reply:
All frontend developers are just human versions of google translate